anesthetic/plot.py

```diff
@@ -362,10 +363,6 @@ def make_2d_axes(params, **kwargs):
                 for a in ax_:
                     a.tick_params('x', bottom=False, top=False,
                                   labelbottom=False, labeltop=False)
-            else:
-                raise ValueError(
-                    "ticks=%s was requested, but ticks can only be one of "
-                    "['outer', 'inner', None]." % ticks)
```

Comment:
I think I agree that this can be safely deleted, and the new test seems to confirm.

Reply:
I was actually trying to actively hit this, and it turned out that it is actually inaccessible...